docs/en/source/buildScorpio.rst

```diff
@@ -142,7 +152,7 @@ Those can be overwriten via the command line or in the docker case as described
 Building Scorpio from source
 ****************************
 
-Scorpio is developed in Java using SpringCloud as microservice framework
+Scorpio is developed in Java using Spring Boot and Quarkus as microservice framework
```

Comment:
Earlier Scorpio was developed in java using Spring Boot but now Scorpio have fully migrated to Quarkus framework.

Reply:
Done